freqtrade/misc.py

```diff
@@ -131,3 +132,15 @@ def deep_merge_dicts(source, destination):
             destination[key] = value
 
     return destination
+
+
+def timeframe_to_seconds(ticker_interval: str) -> int:
+    return Exchange.parse_timeframe(ticker_interval)
```

Comment:
The only think i would like to see is a docstring for these explaining that it takes the ticker_interval in the format "5m", "1h" ...

it's none-obvious and i had to look it up in the ccxt sourcecode to make sure ...  

Rest looks good to me :+1:

Reply:
Done.